services/viva-ms/serverless.yml

```diff
@@ -75,3 +75,19 @@ functions:
               - bankId.collect
             detail-type:
               - BankIdCollectComplete
+
+  syncWorkflow:
+    handler: lambdas/syncWorkflow.main
+    events:
+      - eventBridge:
+          pattern:
+            source:
+              - cases.database
+            detail-type:
+              - MODIFY
+            detail:
+              dynamodb:
+                NewImage:
+                  status:
+                    S: ["ongoing", "submitted"]
```

Comment:
This, will need to be changed. Since the status attribute in dynamoDB will no longer be a string. Suggestion is to wait for the status PR before merging this PR

Reply:
I'm on hold!

Reply:
Fixed the pattern, this should be good to merge. But feel free to test the event on your env also! :)